privacyidea/lib/tokens/hotptoken.py

```diff
@@ -213,6 +253,15 @@ def get_init_detail(self, params=None, user=None):
                     
         return response_detail
 
+    def _get_twostep_parameters(self):
+        """
+        :return: A dictionary with the keys ``2step_salt``,
+        ``2step_difficulty``, ``2step_output``, mapping each key to an integer.
+        """
+        return {'2step_salt': int(self.get_tokeninfo('2step_clientsize')),
+                '2step_output': int(keylen[self.hashlib]),
+                '2step_difficulty': int(self.get_tokeninfo('2step_difficulty'))}
```

Comment:
Will this always be ``int``? Probably in case of hotp and pbkdf2.

Reply:
Hm yeah, I suppose the difficulty will be numeric for any KDF we might use in the future.